win32/src/win32crypt/PyCRYPTHASH.cpp

```diff
@@ -217,9 +216,10 @@ PyObject *PyCRYPTHASH::PyCryptVerifySignature(PyObject *self, PyObject *args, Py
         return NULL;
     if (!PyWinObject_AsHCRYPTKEY(obhcryptkey, &hcryptkey, FALSE))
         return NULL;
-    if (!PyWinObject_AsReadBuffer(obsig, (void **)&pbSignature, &dwSigLen, FALSE))
+    BufferView pybuf(obsig);
+    if (!pybuf.ok())
         return NULL;
-    if (CryptVerifySignature(hcrypthash, pbSignature, dwSigLen, hcryptkey, sDescription, dwFlags)) {
+    if (CryptVerifySignature(hcrypthash, (BYTE*)pybuf.ptr(), pybuf.len(), hcryptkey, sDescription, dwFlags)) {
```

Comment:
style nit: please add a space in `BYTE*`

Reply:
solved by updated PR